demo/server/start_runtime.py

```diff
@@ -38,5 +38,5 @@
 
 alog.configure(default_level="debug")
 
-# Load the model(s) in a gRPC server
-grpc_server.main()
+# Start the Caikit runtime loading the model
+main()
```

Comment:
@hickeyma curious why we wanted to switch it from `grpc_server.main` `__main__.main` ?

Ref: https://github.com/caikit/caikit/blob/main/caikit/runtime/grpc_server.py#L277C5-L277C9 this could also work right?

Reply:
Because that won't start the HTTP. server. See https://github.com/caikit/caikit/blob/main/caikit/runtime/__main__.py#L15

Reply:
ah, yes, got it.